examples/gcoap/client.c

```diff
@@ -287,6 +318,12 @@ int gcoap_cli_cmd(int argc, char **argv)
         ((argc == apos + 2 ||
           argc == apos + 3) && (code_pos > 1))) {     /* post or put */
 
+        /* get unproxied endpoint from address string */
+        if (_addrstr2remote(argv[apos], &remote)) {
+            printf("invalid address\n");
```

Comment:
How about this? More verbose and easier to comprehend for newcomers. (I imagine people confusing addresses, uris, hostnames etc., or messing up the order of the cli arguments. This error points that out.)
```suggestion
            printf("'%s' is not a valid address\n", argv[apos]);
```

Reply:
Sure. After all I added that output to ease understanding of the error :)